### PR TITLE
Update error messages inline with cli commands

### PR DESCRIPTION
**Pull request by prankulmahajan**

### CODE REVIEW

**Comment by contsto** [comment]
:thumbsup: Coverage remained same at [92.3265%]

File: common/messages/messages_en.go [hunk 1]
```diff
@@ -49,42 +49,42 @@ var messagesEn = map[string]util.Message{
 		Description: "The volume ID '%d' could not be deleted from your VPC.",
 		Type:        util.DeletionFailed,
 		RC:          500,
-		Action:      "Verify that the volume ID exists. Run 'ibmcloud is volumes' to list available volumes in your account. If the ID is correct, try to delete the volume with the 'ibmcloud is volume-delete' command. ",
+		Action:      "Verify that the volume ID exists. Run 'ibmcloud is shares' to list available volumes in your account. If the ID is correct, try to delete the volume with the 'ibmcloud is volume-delete' command. ",
```
Comment (sameshai) [inline common/messages/messages_en.go:52]: can we change the volume delete command to ibmcloud is share-delete

File: common/messages/messages_en.go [hunk 2]
```diff
@@ -49,42 +49,42 @@ var messagesEn = map[string]util.Message{
...
-		Action:      "Verify that the specified volume exists. Run 'ibmcloud is volumes' to list available volumes in your account.",
+		Action:      "Verify that the specified volume exists. Run 'ibmcloud is shares' to list available volumes in your account.",
 	},
 	"AccessPointWithAPIDFindFailed": {
 		Code:        AccessPointWithAPIDFindFailed,
 		Description: "No volume access point could be found for the specified volume ID '%s' and access point ID '%s'",
 		Type:        util.VolumeAccessPointFindFailed,
 		RC:          400,
-		Action:      "Verify that a volume access point for your volume exists.Check if volume ID and access point ID is valid",
+		Action:      "Verify that a volume access point for your volume exists. Run `ibmcloud is share-targets <VOLUME_ID>` to list all mount targets for a file share. Check if volume ID and access point ID is valid",
```
Comment (sameshai) [inline common/messages/messages_en.go:80]: If this message is displayed to user can we change this to Run `ibmcloud is share-targets <VOLUME_ID>` to list all volume access point for a volume.

File: common/messages/messages_en.go [hunk 3]
```diff
@@ -49,42 +49,42 @@ var messagesEn = map[string]util.Message{
...
 	},
 	"AccessPointWithVPCIDFindFailed": {
 		Code:        AccessPointWithVPCIDFindFailed,
 		Description: "No volume access point could be found for the specified volume ID '%s' and VPC ID %s",
 		Type:        util.VolumeAccessPointFindFailed,
 		RC:          400,
-		Action:      "Verify that a volume access point for your volume exists.Check if volume ID and VPC ID is valid",
+		Action:      "Verify that a volume access point for your volume exists. Run `ibmcloud is share-targets <VOLUME_ID>` to list all mount targets for a file share. Check if volume ID and VPC ID is valid",
```
Comment (sameshai) [inline common/messages/messages_en.go:87]: Same here

File: common/messages/messages_en.go [hunk 4]
```diff
@@ -188,7 +188,7 @@ var messagesEn = map[string]util.Message{
 		Code:        VolumeAccessPointExist,
 		Description: "The volume ID '%s' could not be deleted from your VPC. Volume has access points which needs to deleted.Please go through the list of VPCs = '%v'",
 		Type:        util.DeletionFailed,
-		Action:      "User need to review all access points and delete them first before deleting volume.", //TODO once cli will be ready then we need to update this error message
+		Action:      "User need to review all access points and delete them first before deleting volume. Run `ibmcloud is share-targets <VOLUME_ID>` to get the mount targets for the file share. ",
```
Comment (sameshai) [inline common/messages/messages_en.go:191]: same here

**Comment by contsto** [comment]
:thumbsup: Coverage remained same at [92.3265%]

File: common/messages/messages_en.go [hunk 5]
```diff
@@ -42,49 +42,49 @@ var messagesEn = map[string]util.Message{
...
-		Action:      "Verify that the specified volume exists. Run 'ibmcloud is volumes' to list available volumes in your account.",
+		Action:      "Verify that the specified volume exists. Run 'ibmcloud is shares' to list available volumes in your account.",
 	},
 	"AccessPointWithAPIDFindFailed": {
 		Code:        AccessPointWithAPIDFindFailed,
 		Description: "No volume access point could be found for the specified volume ID '%s' and access point ID '%s'",
 		Type:        util.VolumeAccessPointFindFailed,
 		RC:          400,
-		Action:      "Verify that a volume access point for your volume exists.Check if volume ID and access point ID is valid",
+		Action:      "Verify that a volume access point for your volume exists. Run `ibmcloud is share-targets <VOLUME_ID>` to list all volume access point for a file share. Check if volume ID and access point ID is valid",
```
Comment (sameshai) [inline common/messages/messages_en.go:80]: Run `ibmcloud is share-targets <VOLUME_ID>` to list all volume access point for a volume.

File: common/messages/messages_en.go [hunk 6]
```diff
@@ -42,49 +42,49 @@ var messagesEn = map[string]util.Message{
...
 	},
 	"AccessPointWithVPCIDFindFailed": {
 		Code:        AccessPointWithVPCIDFindFailed,
 		Description: "No volume access point could be found for the specified volume ID '%s' and VPC ID %s",
 		Type:        util.VolumeAccessPointFindFailed,
 		RC:          400,
-		Action:      "Verify that a volume access point for your volume exists.Check if volume ID and VPC ID is valid",
+		Action:      "Verify that a volume access point for your volume exists. Run `ibmcloud is share-targets <VOLUME_ID>` to list all volume access point for a file share. Check if volume ID and VPC ID is valid",
```
Comment (sameshai) [inline common/messages/messages_en.go:87]: Run ibmcloud is share-targets <VOLUME_ID> to list all volume access point for a volume.

File: common/messages/messages_en.go [hunk 7]
```diff
@@ -188,7 +188,7 @@ var messagesEn = map[string]util.Message{
 		Code:        VolumeAccessPointExist,
 		Description: "The volume ID '%s' could not be deleted from your VPC. Volume has access points which needs to deleted.Please go through the list of VPCs = '%v'",
 		Type:        util.DeletionFailed,
-		Action:      "User need to review all access points and delete them first before deleting volume.", //TODO once cli will be ready then we need to update this error message
+		Action:      "User need to review all access points and delete them first before deleting volume. Run `ibmcloud is share-targets <VOLUME_ID>` to get the volume access point for the file share. ",
```
Comment (sameshai) [inline common/messages/messages_en.go:191]: Run ibmcloud is share-targets <VOLUME_ID> to list all volume access point for a volume.

**Review by sameshai — CHANGES_REQUESTED** [review]

File: common/messages/messages_en.go [hunk 8]
```diff
@@ -35,160 +35,160 @@ var messagesEn = map[string]util.Message{
...
+		Description: "A file share with the specified file share name '%s' does not exist.",
 		Type:        util.RetrivalFailed,
 		RC:          404,
-		Action:      "Verify that the specified volume exists. Run 'ibmcloud is volumes' to list available volumes in your account.",
+		Action:      "Verify that the specified file share exists. Run 'ibmcloud is shares' to list available file shares in your account.",
 	},
 	"AccessPointWithAPIDFindFailed": {
 		Code:        AccessPointWithAPIDFindFailed,
-		Description: "No volume access point could be found for the specified volume ID '%s' and access point ID '%s'",
+		Description: "No mount target could be found for the specified file share ID '%s' and mount point ID '%s'",
```
Comment (sameshai) [inline common/messages/messages_en.go:77]: mount point ID --> moutn target ID

File: common/messages/messages_en.go [hunk 9]
```diff
@@ -35,160 +35,160 @@ var messagesEn = map[string]util.Message{
...
 		Type:        util.VolumeAccessPointFindFailed,
 		RC:          400,
-		Action:      "Verify that a volume access point for your volume exists.Check if volume ID and access point ID is valid",
+		Action:      "Verify that a mount point for your file share exists. Run `ibmcloud is share-targets <SHARE_ID>` to list all mount targets for a file share. Check if file share ID and mount target ID is valid",
```
Comment (sameshai) [inline common/messages/messages_en.go:80]: mount point  --> mount target

File: common/messages/messages_en.go [hunk 10]
```diff
@@ -35,160 +35,160 @@ var messagesEn = map[string]util.Message{
...
+		Description: "No mount target could be found for the specified file share ID '%s' and VPC ID %s",
 		Type:        util.VolumeAccessPointFindFailed,
 		RC:          400,
-		Action:      "Verify that a volume access point for your volume exists.Check if volume ID and VPC ID is valid",
+		Action:      "Verify that a mount target for your file share exists. Run `ibmcloud is share-targets <SHARE_ID>` to list all volume targets for a file share. Check if file share ID and VPC ID is valid",
 	},
 	"CreateVolumeAccessPointFailed": {
 		Code:        CreateVolumeAccessPointFailed,
-		Description: "The volume ID '%s' could not create access point for VPC ID %s.",
+		Description: "The file share ID '%s' could not create mount taregt for VPC ID %s.",
```
Comment (sameshai) [inline common/messages/messages_en.go:91]: typo for target

**Review by sameshai — APPROVED** [review]
LGTM

**Comment by contsto** [comment]
:thumbsup: Coverage remained same at [92.5417%]

**Review by arahamad-zz — APPROVED** [review]
lgtm

**Pull request merged**